Lib/lib2to3/pgen2/tokenize.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006 Python Software Foundation.
+# Copyright (c) 2001, 2002, 2003, 2004, 2005, 2017 Python Software Foundation.
```

Comment:
This change is wrong. You should either add all years, or leave things as they are.